project/Build.scala

```diff
@@ -1778,6 +1780,21 @@ object Build {
       previousArtifactSetting,
       mimaBinaryIssueFilters ++= BinaryIncompatibilities.Library,
 
+      /* Silence a Scala 2.13.13+ warning that we cannot address without breaking our API.
+       * See `js.WrappedDictionary.keys` and `js.WrappedMap.keys`.
```

Comment:
It goes with these changes in the build.

Reply:
Instead of this, have you considered making a compat annotation for `scala.annotation.nowarn`? IIUC we could achieve the same, but it would stay local.

Reply:
It would inevitably leak into our public API. I don't think that's a good idea.

Reply:
Thinking more about this, I don't see *how* we would even do that. It would require patch-version-dependent source files. It's not really something we can do in our `library`, since we only publish one version per *minor* Scala version. So users of older Scala version would load a `library` that refers to an annotation in `scala-library` that does not actually exist for them. The Scala compiler is supposed to be somewhat resilient to these situations for annotations, but it's still not something I would fundamentally rely on.

Reply:
... `nowarn` seems to indeed be a static annotation... why? :thinking: 

In this case, fair enough.